docs/jobs/ai-assistant.md

```diff
@@ -0,0 +1,60 @@
+---
+title: Using OpenFn with AI Assistant
+sidebar_label: Using AI Assistant
+---
+
+:::warning
+
+AI makes mistakes. Always review the AI's response carefully and make sure to
```

Comment:
Ach. I don't think I'd put this first, or in such a dramatic warning box. Also mistakes aren't the only thing to be concerned about.

I think the current UI covers this stuff well right now. I think if we were to address safety/accuracy in the docs, we should do it in a dedicated section. Maybe you could have a section called "How Accurate Is It?" And we can get into the issues there